frame/ibc/ibc-runtime-api/src/lib.rs

```diff
@@ -8,6 +8,8 @@ use sp_std::vec::Vec;
 sp_api::decl_runtime_apis! {
 	/// IBC Runtime Apis
 	pub trait IbcRuntimeApi {
+		/// Get parachain id
+		fn para_id() -> u32;
```

Comment:
`cumulus-pallet-parachain-info` already provides this

Reply:
@Wizdave97

Reply:
Yeah, but there's no runtime api to get the parachain id, so I'm providing it to the rpc handler through this runtime api